package.json

```diff
@@ -393,6 +393,106 @@
                 "key": "ctrl+p",
                 "when": "listFocus && !inputFocus"
             },
+            {
+                "key": "j",
```

Comment:
So this is for navigating lists with j/k?

Reply:
Yes, it is. Taken from VSCodeVim.